dll/win32/shell32/utils.cpp

```diff
@@ -99,3 +99,151 @@ SHFindComputer(LPCITEMIDLIST pidlRoot, LPCITEMIDLIST pidlSavedSearch)
 
     return SUCCEEDED(hr);
 }
+
+static HRESULT Int64ToStr(LONGLONG llValue, LPWSTR pszValue, UINT cchValue)
+{
+    WCHAR szBuff[40];
+    UINT ich = 0, ichValue;
+#if (WINVER >= _WIN32_WINNT_VISTA)
```

Comment:
ok with WINVER, prefer using e.g. WINVER_VISTA (OK I know the values usually match with the WIN32_WINNT_xxx defines...)